### PR TITLE
[zh-cn] update devicemotion event

### PR DESCRIPTION
**Pull request by jasonren0403**

### CODE REVIEW

File: files/zh-cn/web/api/window/devicemotion_event/index.md [hunk 1]
```diff
@@ -1,23 +1,56 @@
...
 slug: Web/API/Window/devicemotion_event
+l10n:
+  sourceCommit: 6d4f585b94068dc0dfd733047fb2229dca38b1eb
 ---
 
-{{ ApiRef() }}
+{{APIRef("Device Orientation Events")}}{{securecontext_header}}
 
-## 摘要
+**`devicemotion`** 事件每隔一定时间触发一次，显示设备当时受到的物理加速度力的大小。如果有的话，它还会提供有关旋转速率的信息。
```
Comment (yin1999) [inline files/zh-cn/web/api/window/devicemotion_event/index.md:10]: I'm not sure about the English content: why there is a word "force". Could you check this?

```suggestion
**`devicemotion`** 事件每隔一定时间触发一次，显示设备当时受到的物理加速度的大小。如果有的话，它还会提供有关旋转速率的信息。
```

File: files/zh-cn/web/api/window/devicemotion_event/index.md [hunk 2]
```diff
@@ -1,23 +1,56 @@
...
+
+## 事件属性
 
-这里的`funcRef`是一个函数的引用。这个函数接收一个{{ domxref("DeviceMotionEvent") }} 对象类型的参数描述发生的动作。
+- {{DOMxRef("DeviceMotionEvent.acceleration")}} {{ReadOnlyInline}}
+  - : 给出设备在 x、y、z 三轴上的加速度。加速度用 [m/s²](https://zh.wikipedia.org/wiki/米每二次方秒) 表示。
+- {{DOMxRef("DeviceMotionEvent.accelerationIncludingGravity")}} {{ReadOnlyInline}}
+  - : 在重力作用下，给出设备在 x、y、z 三个轴上的加速度的对象。加速度单位为 [m/s²](https://zh.wikipedia.org/wiki/米每二次方秒)。
+- {{DOMxRef("DeviceMotionEvent.rotationRate")}} {{ReadOnlyInline}}
+  - : 一个给出设备在三个方向轴（阿尔法轴、贝塔轴和伽马轴）上方向变化率的对象。旋转率以每秒度数表示。
```
Comment (yin1999) [inline files/zh-cn/web/api/window/devicemotion_event/index.md:37]:
```suggestion
  - : 一个给出设备绕三个方向轴（阿尔法轴、贝塔轴和伽马轴）的旋转速率的对象。旋转速率以度每秒表示。
```

File: files/zh-cn/web/api/window/devicemotion_event/index.md [hunk 3]
```diff
@@ -29,6 +62,5 @@ Where `funcRef` is a reference to a function. This function receives a {{ domxre
 
 ## 参考
```
Comment (yin1999) [inline files/zh-cn/web/api/window/devicemotion_event/index.md:63]:
```suggestion
## 参见
```

File: files/zh-cn/web/api/window/devicemotion_event/index.md (hunk 1 above)
Comment (jasonren0403) [reply files/zh-cn/web/api/window/devicemotion_event/index.md:10]: > I'm not sure about the English content: why there is a word "force". Could you check this?

I found it somehow confusing too 🤔

Comment (yin1999) [reply files/zh-cn/web/api/window/devicemotion_event/index.md:10]: mdn/content#32153

**Review by yin1999 — APPROVED** [review]
Thank you @jasonren0403

**Pull request merged**